src/main/scala/inox/solvers/smtlib/SMTLIBTarget.scala

```diff
@@ -503,6 +511,16 @@ trait SMTLIBTarget extends SMTLIBParser with Interruptible with ADTManagers {
       case (SHexadecimal(h), Some(CharType)) =>
         CharLiteral(h.toInt.toChar)
 
+/*
+ *      TODO
+ *       case (FixedSizeBitVectors.BitVectorConstant(n, b), Some(BVType(size))) if b == size =>
+ *         BVLiteral(n, size)
+ *
+ *       case (SHexadecimal(h), Some(BVType(size))) if size <= 32 =>
+ *         BVLiteral(BigInt(h.toInt), size)
+ *       // FIXME SHexadecimal has no conversion for integer larger than 32 bits.
+ */
+
```

Comment:
Not sure if this is useful or not. It was added into Leon but maybe no longer required for inox/stainless?